app/views/messages/_right_column.html.erb

```diff
@@ -31,46 +31,42 @@
   </div>
 
   <!-- Only one of these headers is visible at a time, depending on width -->
-  <div id="wide-header" class="flex-none justify-between ml-2 mt-2 mb-3 h-10 text-black dark:text-white text-lg hidden md:flex">
+  <div id="wide-header" class="justify-between flex-none hidden h-10 mt-2 mb-3 ml-2 text-lg text-gray-950 dark:text-gray-100 md:flex">
     <%= render 'messages/right_column/assistant_header' %>
-
-    <%= link_to new_assistant_message_path(@assistant), class: "block cursor-pointer inline-flex mr-3 px-2 py-2 group", data: { role: "pencil" } do %>
-      <%= icon "pencil-square", variant: :outline, size: 22, class: 'stroke-2 text-zinc-600 group-hover:text-zinc-900 dark:text-zinc-400 dark:group-hover:text-white', title: "New chat", tooltip: :left %>
-    <% end %>
   </div>
```

Comment:
Let's keep the pencil in the top-right of the wide header. You were probably just comparing our design to chatgpt on the web. But annoyingly, the design of ChatGPT on mobile (phone & tablet) is slightly different than the web. On mobile, they have a persistent "new" button in the top-right.

For ChatGPT, they clearly have a separate codebase (and maybe separate team) doing the mobile stuff, so they've diverged. But when we get mobile working it'll be the exact same CSS. So over time we'll incorporate more of the mobile-specific stuff into our overall design so that we don't have to target those two environments too much.